resources/suite-runner.mjs

```diff
@@ -108,8 +111,10 @@ export class SuiteRunner {
         if (this.#suite === WarmupSuite)
             return;
 
-        const total = syncTime + asyncTime;
-        this.#suiteResults.tests[test.name] = { tests: { Sync: syncTime, Async: asyncTime }, total: total };
+        let total = syncTime + asyncTime;
+        if (this.#params.measurePrepare)
+            total += this.#prepareTime;
```

Comment:
I think if we did this, we should probably refuse to compute/show the final score. It's going to be very misleading otherwise. We don't want people to start reporting score with prepare time included as some kind of Speedometer score variant.